<ui version="4.0" >
 <class>DiscretizationInputForm</class>
 <widget class="QWidget" name="DiscretizationInputForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_fileLabel" >
          <property name="text" >
           <string>Filename:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="_filename" />
        </item>
        <item>
         <widget class="QPushButton" name="browseButton" >
          <property name="text" >
           <string>Browse ...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_unitsLabel" >
          <property name="text" >
           <string>Unit conversion for file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="_unitsComboBox" />
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_unitConversion" >
          <property name="decimals" >
           <number>20</number>
          </property>
          <property name="value" >
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_coordinateSystemLabel" >
          <property name="text" >
           <string>Coordinate system for file:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="_coordinateSystemName" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="coordinateSystem" >
          <property name="text" >
           <string>Choose...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="_originLabel" >
          <property name="text" >
           <string>Offset in coordinate system:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_x" />
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_y" />
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="_z" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
